dashboard/config.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+ENV = os.getenv("FLASK_ENV")
+SERVER_NAME = os.getenv("SERVER_NAME")
+DEBUG = ENV == "development"
```

Comment:
[Flask already does this, we shouldn't interfere](https://flask.palletsprojects.com/en/1.1.x/config/#environment-and-debug-features)
>The ENV and DEBUG config values are special because they may behave inconsistently if changed after the app has begun setting up. In order to set the environment and debug mode reliably, Flask uses environment variables.


```suggestion
```